File: C++/Contoh Looping/Uncounted Loop/Do While.cpp
#include <iostream>
using namespace std;

int main(){
	
	char ulang = 'y';
	int i = 0;
	
	do {
	cout << " " << endl;
	cout << "Masukkan huruf y untuk contoh perulangan" << endl;
	cout << " " << endl;
	cout << "Masukkan huruf : ";
	cin >> ulang;
	i++;
	}while(ulang == 'y');
	
	cout << " " << endl;
	cout << "Selesai" << endl;

	return 0;
}

File: C#/Contoh Looping/Uncounted Loop/Do While.cs
using System;

namespace Main
{
    class main
    {
        static void Main(string[] args)
        {
            char ulang = 'y';
            int i = 0;

            do
            {
                Console.WriteLine();
                Console.WriteLine("Masukkan huruf y untuk contoh perulangan");
                Console.WriteLine();
                Console.Write("Masukkan huruf : ");
                ulang = char.Parse(Console.ReadLine());
                i++;
            } while (ulang == 'y');

            Console.WriteLine();
            Console.WriteLine("Selesai");

        }
    }
}
